core/docs/spark-qualification-tool.md

```diff
@@ -205,6 +205,9 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
                                      min(minute),h(hours),d(days),w(weeks),m(months).
                                      If a period is not specified it defaults to
                                      days.
+      --spark-env  <arg>...          Spark environment where CPU workloads were
+                                     executed.  Options include onprem, dataproc,
+                                     and emr.  Default is onprem.
```

Comment:
`spark-env` gives me impression it is something related to Spark conf/properties; while it is more like the cluster-deployment/platform.
Should we name this something different like `platform`, `cloud-platform` `deploy-env`, ..etc?

Reply:
1. Good idea on changing spark-env to something more intuitive.  I'll switch it to platform.
2. For right now, we don't have different exec/expression support per environment as it's universal support in our plugin irrespective of environment.  
3. That shouldn't change as the original operatorsScore.csv remains unchanged.